ci-scripts/snapcraft.sh

```diff
@@ -5,10 +5,13 @@ set -exuo pipefail
 
 tar xf ./artifacts/${NITROKEY_APP_BUILD_ARTIFACT_VERSION}.tar.gz
 
+echo Git Version:
+echo $(git describe --always)
+
 mkdir -p artifacts
 OUTDIR="$(realpath artifacts)"
 
-OUTNAME="nitrokey-app_${NITROKEY_APP_BUILD_ARTIFACT_VERSION}_amd64.snap"
```

Comment:
Looking as a whole, it would be better to overwrite the variable [1] at the beginning of the script, instead of replacing it with the direct calls. Better solution would be to create a new one in the metadata script, which would be adjusted to snapcraft only, and do not decide here about that.

[1] `NITROKEY_APP_BUILD_ARTIFACT_VERSION`